spec/abci/abci++_methods.md

```diff
@@ -452,7 +362,11 @@ title: Methods
       that the `RequestPrepareProposal.max_tx_bytes` limit is respected by those transactions
       returned in `ResponsePrepareProposal.txs` .
     * As a result of executing the prepared proposal, the Application may produce block events or transaction events.
-      The Application must keep those events until a block is decided. It will then forward the events to the `BeginBlock-DeliverTx-EndBlock` functions depending on where each event should be placed, thereby returning the events to CometBFT.
+      The Application must keep those events until a block is decided and then pass them on to CometBFT via
+      `ResponseFinalizeBlock`.
+<!--
+ TODO CHECK THIS 
+-->
```

Comment:
I understand your hesitation.
We modified this late in the `v0.37.x` process following SDK's problems with the previous interface, so we won't change it now.
Conclusion, we need to keep the text that says that there are no additional checks.